### PR TITLE
User sessions

### PR DESCRIPTION
**Pull request by vicper-4**
sing_up routes incorrectly to Customer edit page when errors occur. The password and password_confirmation does not seem to be checked against each other.

### CODE REVIEW

**Comment by vicper-4** [comment]
[310. Registrering [60][4]](https://trello.com/c/JTYFvi0X/34-310-registrering-60-4)

**Comment by vicper-4** [comment]
signing works now and can be checked for with logged_in?. the user is accessible by current_user. 
Loging out does not seem to work, and the sign_up page is still not working as intended

File: Rails_app/app/helpers/sessions_helper.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+module SessionsHelper
+  def log_in(customer)
+    session[:customer_id] = customer.id
+  end
+
+  def log_out
```
Comment (vicper-4) [inline Rails_app/app/helpers/sessions_helper.rb]: seemes to be something wrong here

**Comment by vicper-4** [comment]
Logout fixed.

**Comment by vicper-4** [comment]
[309. Kundinlogg [50][2]](https://trello.com/c/jNnS2y6N/33-309-kundinlogg-50-2)

**Comment by nikalas** [comment]
Tycker det ser bra ut men kan inte köra branchen, så kan inte testa. Har kört bundle install och db:migrate men får ändå fel på:

> LoadError in FrontendPagesController#index
> cannot load such file -- bcrypt
> Extracted source (around line #16):
> ```
> 14                    allow_blank: true
> 15
> 16  has_secure_password
> 17  validates :password,  confirmation: true,
> 18                       :presence => { :on => :create },
> 19                       length: { minimum: 6 }
> 
> ```

Vet du varför det blir så?

**EDIT:** Så, det visar sig att det kan vara bra att starta om servern när man installerar nya gems. Surprise!

**Comment by vicper-4** [comment]
bcryp behövde en password?digetst istället för password i users, men db:migrate räckte för mig

**Review by nikalas — APPROVED** [review]

**Pull request merged**